### ISSUE TITLE
整个十月多少人无法使用。十一月照常扣使用日期

### ISSUE THREAD

**Issue by Restlessness** [getlantern/forum#6492]
不能使用。也没说个抱歉。月数照扣，蓝灯是真伟大

**Comment by Sincher** [comment]
蓝灯一直的品质是可以的，而且也不算贵，翻墙软件本来就有这风险。这个月波动看看各位的嘴脸。

**Comment by Restlessness** [comment]
我买的ss十月不稳定。补偿了一个月。 客服都是两小时内回复疑问。 Express VPN我买了一年。客服随叫随到。express VPN邀请一个人注册送30天。我现在获得了1320天。蓝灯邀请一个获得三个月。我获得六个月。

**Comment by Restlessness** [comment]
他们做的不好,他们一点错没有？我们等是应该的,有质疑就是恶嘴脸@Sincher你他妈还真是圣母婊。

**Comment by xiguataida** [comment]
@Sincher  这大概就是给屎就吃屎，给尿就喝尿，不知道说声想喝水的蠢材吧

**Comment by bestlx001** [comment]
@Sincher 你这种人就是去饭店花了1000块吃大餐，结果吃到了老鼠屎，然后就跟饭店说没事，你们家菜一直都很好，吃饭本来就有风险吃到屎，这次偶尔发生也是应该的。说完你就把屎吃了

**Comment by sjm0240111** [comment]
我才买一个星期然后不能用了，真是B了狗，能退款么？

**Comment by jasonzhao1002** [comment]
确实 昨天买了一个月Express VPN  整体很不错 720p视频无压力  虽然一个月比较贵 86 但是比蓝灯稳定很多 要有兴趣的话 点这个邀请可以多给你一个月 http://www.expressrefer.com/refer-friend?referrer_id=14546108&utm_campaign=referrals&utm_medium=copy_link&utm_source=referral_dashboard  当然 我自己也多一个月

**Comment by jyufeng** [comment]
截至今天，此刻4.4.0版速度上来了。但如楼主说的，lantern官方得出来说个话，是不是应该给予1-2个月的赠时？在此时间段萤火虫、赛风、自由门等免费软件都ok，但独独付费的lantern不好使，也是天朝奇葩。

**Comment by lemonskkyt** [comment]
@jyufeng 我大会结束就一直不能用，然后赛风只有昨天挂了下，真不知道lantern的技术都在忙些什么，说实话，lantern 的服务是真心差，去年7月初刚开始收费我就买了，然后买来第一天就有问题，系统会把付费用户误判成免费用户，然后限速到只有30多KB/S（大概，记得不是很清楚了），这才刚收费，你都免费了几年，收费系统出问题接着免费几天，满足付费用户的使用需求不难吧，结果lantern是怎么干的？继续限速，然后花了5 6天修复，反正他们眼里，服务质量、客户满意度都不重要

**Comment by jyufeng** [comment]
@lemonskkyt嗯，lantern这服务质量，态度具有天朝的天然基因？
